docs/diagnostic-logs.md

```diff
@@ -35,34 +44,108 @@ Diagnostic logs are disabled by default. To enable diagnostic logs, follow these
 
     ![Pane navigation to diagnostic settings](./images/diagnostic-logs/diagnostic-settings-menu-item.png)
 
-1.	Then click **Add diagnostic setting**.
+1.	Then you will get a full view of the diagnostic settings.
 
-	![Add diagnostic logs](./images/diagnostic-logs/add-diagnostic-setting.png)
+	![Diagnostic settings' full view](./images/diagnostic-logs/azure-signalr-diagnostic-settings.png)
 
-1.	Set the archive target that you want. Currently, we support **Archive to a storage account** and **Send to Log Analytics**.
+1. Configure the log source settings.
+   1. In **Log Source Settings** section, check the the specific log type you want to collect for all connections. Otherwise the the log will be collected only for [diagnostic clients](#diagnostic-client).
```

Comment:
Since there is only one item, no need to be a list?

Reply:
I make it 2 items. If I unformat the list, the indentation will be wierd.